src/components/theme/Logout/Logout.jsx

```diff
@@ -55,7 +55,13 @@ class Logout extends Component {
    * @returns {string} Markup for the component.
    */
   render() {
-    return <Login location={{ query: this.props.location.query }} />;
+    console.log(this.props.location);
```

Comment:
@sumukhah Please remove this console.logs.

Reply:
Thank you for review @nileshgulia1 ,
will fix it soon,